src/main/java/dan200/computercraft/core/lua/LuaJLuaMachine.java

```diff
@@ -412,6 +415,7 @@ public void execute()
                                             }
                                             catch( Throwable t )
                                             {
+                                                ComputerCraft.log.error( "Error running task", t);
```

Comment:
I'm happy to switch to trace or debug (neither of which are shown IIRC). I chose error as this being hit indicates a bug with someone's code, which trace does not. I'd argue it should be noisy though - even when an error won't crash Minecraft (such as in network code), it'll still log it with stack trace and everything.